DepotDownloader/ContentDownloader.cs

```diff
@@ -590,7 +590,7 @@ static DepotDownloadInfo GetDepotInfo(uint depotId, uint appId, ulong manifestId
             if (manifestId == INVALID_MANIFEST_ID)
             {
                 manifestId = GetSteam3DepotManifest(depotId, appId, branch);
-                if (manifestId == INVALID_MANIFEST_ID && branch != "public")
+                if (manifestId == INVALID_MANIFEST_ID && branch.ToLower() != "public")
```

Comment:
Better to avoid creating a whole new string if we don't need to.
```suggestion
                if (manifestId == INVALID_MANIFEST_ID && !string.Equals(branch, "public", StringComparison.OrdinalIgnoreCase))
```

Reply:
Most importantly, using `ToLower` instead of `ToLowerInvariant` is going to cause issues for people with specific locales.

Reply:
that's true in general, but I don't think we'd have any variance problems with the fixed string `"public"`?

Reply:
The `i` is often a problem, I didn't see an issue with other characters yet, and not sure if it would cause issue in this specific case, but better being safe than sorry.